srcalc/src/main/webapp/WEB-INF/views/fragments/discreteNumericalFragment.jsp

```diff
@@ -0,0 +1,18 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ taglib uri="/WEB-INF/srcalc.tld" prefix="srcalc" %>
+
+<!-- Wrap both the radio button and numerical entry in a span.radioLabel
+     for proper spacing. -->
+<span class="radioLabel"><form:radiobutton path="${displayItemParam.varPath}" cssClass="numericalRadio" value="numerical"/>
+<c:set var="numericalVarName" value="${displayItemParam.key}$numerical" />
+<c:set var="numericalVarPath" value="${srcalc:dynamicValuePath(numericalVarName)}" />
```

Comment:
Simplification: you could provide `DiscreteNumericalVariableView.getNumericalVarPath()`.